index.js

```diff
@@ -182,3 +183,45 @@ function LoadProjects() {
 }
 
 document.addEventListener('DOMContentLoaded', LoadProjects, false);
+
+const form = document.getElementById('contactForm');
+const emailFormat = 'Please enter a valid email adress';
```

Comment:
- [ ] [REQUIRED] You have done a great implementation, however, one of the requirements states that the error message should show up close to the `submit` button but you have yours in between the email field and the testarea. Kindly fix this.

![Screenshot from 2022-03-02 17-29-57](https://user-images.githubusercontent.com/58692590/156405673-3fceb2f4-773c-489f-aa16-5da1d31943ff.png)